palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/web/PalisadeController.java

```diff
@@ -52,6 +55,15 @@ public DataRequestResponse registerDataRequestSync(@RequestBody final RegisterDa
         return response;
     }
 
+    @PostMapping(value = "/registerDataRequest/{userId}/{resourceId}", consumes = "application/json", produces = "application/json")
+    public DataRequestResponse registerDataRequestSyncRestful(@PathVariable final String userId, @PathVariable final String resourceId) {
+        RegisterDataRequest request = new RegisterDataRequest()
+                .userId(new UserId().id(userId))
+                .resourceId(resourceId)
+                .context(new Context().purpose(""));
+        return registerDataRequestSync(request);
+    }
```

Comment:
Do we need this? if so then the purpose should be parameterised at the very least or even better have the whole context passed in

Reply:
removed